drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -107,13 +107,22 @@ function DragNDropTemplates(url_name) {
         var item_content = h('div', { innerHTML: item_content_html, className: "item-content" });
         if (item.is_placed) {
             // Insert information about zone in which this item has been placed
-            var item_description_id = url_name + '-item-' + item.value + '-description';
+            var item_description_id = configuration.url_name + '-item-' + item.value + '-description';
             item_content.properties.attributes = { 'aria-describedby': item_description_id };
             var zone_title = (zone.title || "Unknown Zone");  // This "Unknown" text should never be seen, so does not need i18n
+            var description_content;
+            if (configuration.mode === DragAndDropBlock.ASSESSMENT_MODE) {
+              // In assessment mode placed items will "stick" even when not in correct zone.
+              description_content = gettext('Placed in: ') + zone_title;
```

Comment:
@cahrens Thanks for pointing that out. I switched to interpolation in this line and in old code a couple of lines below.